semanticz-connector-neptune/src/test/java/zone/cogni/semanticz/connectors/neptune/NeptuneRdfStoreServiceTest.java

```diff
@@ -0,0 +1,244 @@
+package zone.cogni.semanticz.connectors.neptune;
+
+import org.apache.jena.query.Query;
+import org.apache.jena.query.QueryExecution;
+import org.apache.jena.query.QueryFactory;
+import org.apache.jena.query.ResultSet;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdfconnection.RDFConnection;
+import org.apache.jena.rdfconnection.RDFConnectionRemote;
+import org.apache.jena.rdfconnection.RDFConnectionRemoteBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.junit.jupiter.MockitoExtension;
+import zone.cogni.sem.jena.template.JenaResultSetHandler;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class NeptuneRdfStoreServiceTest {
```

Comment:
Having integration tests for Neptune would be excellent, if feasible and possible - similarly to [GraphDb](https://github.com/cognizone/semanticz-connectors/blob/develop/semanticz-connector-graphdb/src/test/java/zone/cogni/semanticz/connectors/graphdb/GraphdbSparqlServiceTest.java) and Virtuoso. Currently they can be enabled via an env var to be used either locally, or [even on Jenkins](https://jenkins.cogni.zone/job/cz/job/semanticz/job/connectors/job/builds/job/semanticz-connectors/)

Reply:
https://github.com/cognizone/semanticz-connectors/issues/24